octo-board-service/src/main/java/com/revature/octo/board/model/OctoBoard.java

```diff
@@ -0,0 +1,86 @@
+package com.revature.octo.board.model;
+
+import java.util.Date;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinTable;
+
+
+public class OctoBoard {
+	
+	@Id
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="BOARD_ID_SEQ")

```

Comment:
which way are we gonna make sequences? guess wait for patrick to get back to us on that